packages/teams-js/src/public/interfaces.ts

```diff
@@ -1150,6 +1154,22 @@ export interface AppEligibilityInformation {
   userClassification: UserClassification | null;
 }
 
+/**
+ * @hidden
+ * @beta
+ * Represents the feature set available to the user.
+ */
+export interface FeatureSet {
+  /**
+   * Server Feature set
+   */
+  serverFeatures: string[];
+  /**
+   * UX Feature set
+   */
+  uxFeatures: string[];
```

Comment:
Should these be declared as `ReadonlyArray<string>` to ensure they aren't accidentally changed and to express the semantic that they're an "input only" property (e.g. they are immutable properties from the host)?

Reply:
ReadonlyArray makes sense. Updated the code